src/table/utils/handle-accessibility.ts

```diff
@@ -79,10 +79,13 @@ export const handleResetFocus = ({
   setFocusedCellCoord,
   keyboard,
   announce,
+  totalsPosition,
 }: HandleResetFocusProps) => {
   updateFocus({ focusType: 'removeTab', cell: findCellWithTabStop(rootElement) });
   // If you have selections ongoing, you want to stay on the same column
-  const cellCoord: [number, number] = isSelectionMode ? [1, focusedCellCoord[1]] : [0, 0];
+  const cellCoord: [number, number] = isSelectionMode
+    ? [totalsPosition === 'top' ? 2 : 1, focusedCellCoord[1]]
+    : [0, 0];
```

Comment:
I would split this up. Nested ternaries are a bit hard to read. Something like this
```suggestion
    const selectionCellCoord = [totalsPosition === 'top' ? 2 : 1, focusedCellCoord[1]];
    const cellCoord: [number, number] = isSelectionMode ? selectionCellCoord : [0, 0];
```